raiden-ts/docs-source/SDK-Development.md

```diff
@@ -226,23 +227,23 @@ action.pipe(
+The SDK tests are located at the [raiden/tests](https://github.com/raiden-network/light-client/tree/master/raiden-ts/tests) subfolder. The testing framework used is [jest](http://jestjs.io), and the complete suite can be run with `yarn test` command in the SDK root folder. This will run both [unit](https://github.com/raiden-network/light-client/tree/master/raiden-ts/tests/unit) and [e2e](https://github.com/raiden-network/light-client/tree/master/raiden-ts/tests/e2e) tests (files ending with `.spec.ts`), and collect coverage in the `raiden/.coverage` folder. Keeping an eye on `raiden/.coverage/lcov-report/index.html` during test writting can be a good guide to writing them, although just covering the lines aren't enough, and some thought must be put into imagining possible scenarios, invariants, critical and edge states, and testing them throughout the respective tests.
 
 ### Unit tests
 
-The unit tests try to cover as much as possible the individual functions, by testing behavior through expected and unexpected inputs, and their respective outputs. For that, we use extensively [jest mocks](https://jestjs.io/docs/en/mock-functions) to contextually replace external and internal dependencies of each tested function, and force the external logic to behave on the different (possible, conceivable) ways and ensure the our tested logic handles all of them. Most of the unit mocks are in [raiden/tests/unit/mocks.ts](https://github.com/raiden-network/light-client/blob/master/raiden/tests/unit/mocks.ts), but some others are needed to be put in the beginning of the respective test files, for the mocking to take place before importing code using the mocked logic.
+The unit tests try to cover as much as possible the individual functions, by testing behavior through expected and unexpected inputs, and their respective outputs. For that, we use extensively [jest mocks](https://jestjs.io/docs/en/mock-functions) to contextually replace external and internal dependencies of each tested function, and force the external logic to behave on the different (possible, conceivable) ways and ensure the our tested logic handles all of them. Most of the unit mocks are in [raiden/tests/unit/mocks.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/unit/mocks.ts), but some others are needed to be put in the beginning of the respective test files, for the mocking to take place before importing code using the mocked logic.
```

Comment:
On this `Unit tests` section, it may be worth describing that we don't call the epics directly anymore, since these tests setup was too cumbersome. Currently, our unit tests are half-way to integration tests (although still as fast and complete as unit tests), since it sets up the whole state machine and subscribe to all epics at once, input/requests being injected in the `store`,  and output being tested/collected in an `output` array, per mocked "client". This new pattern has shown to be orders of magnitude easier to write and more correct to set up, while as fast. individual utils and helpers are still tested independently whenever possible.